### ISSUE TITLE
popup toast doesn't appear if isPresented is .constant(true)

### ISSUE THREAD

**Issue by ronyfadel** [exyte/PopupView#52]
In this code:

```
            .popup(isPresented: .constant(true),
                   type: .toast,
                   position: .bottom,
                   dragToDismiss: true,
                   closeOnTap: false,
                   closeOnTapOutside: true,
                   dismissCallback: {
                   // on dismiss
                  }) {
                CardView { ... }
```

`PopupView` bases the appearAction on `isPresented` changing:

```
        .valueChanged(value: isPresented) { isPresented in
            appearAction(isPresented: isPresented)
        }
```

But if the binding is a `.constant(true)` since PopupView's inception, it doesn't appear.

**Comment by f3dm76** [comment]
Hey @ronyfadel, great catch! Thank you very much for bringing this to my attention. I'm happy to say, I fixed the issue (added onAppear call as well as on change). I added my fix to version 1.0.4, you are most welcome to give it a go. Have a marvellous day!

**Comment by f3dm76** [comment]
Hello @ronyfadel, it's been a while so I'm gonna this issue, but please do let me know if the problem is not fixed. Have a good day!
